docs/M365/MDO/MDOThreatPolicyChecker.md

```diff
@@ -0,0 +1,32 @@
+# MDOThreatPolicyChecker
+ This script checks which Microsoft Defender for Office 365 and Exchange Online Protection threat policies cover a particular user, including anti-malware, anti-phishing, inbound and outbound anti-spam, as well as Safe Attachments and Safe Links policies in case these are licensed for your tenant. In addition, the script can check for threat policies that have inclusion and/or exclusion settings that may be redundant or confusing and lead to missed coverage of users or coverage by an unexpected threat policy.
+
+## Download
+Download the latest release: [MDOThreatPolicyChecker.ps1](https://github.com/microsoft/CSS-Exchange/releases/latest/download/MDOThreatPolicyChecker.ps1)
+
+## Parameters
+
+Parameter | Description |
+----------|-------------|
+CsvFilePath | Allows you to specify a CSV file with a list of email addresses to check. Csv file must include a first line with header Email.
+EmailAddresses | Allows you to specify email address or multiple addresses separated by commas.
+IncludeMDOPolicies | Checks both EOP and MDO (Safe Attachment and Safe Links) policies for user(s) specified in the CSV file or EmailAddresses parameter.
+OnlyMDOPolicies | Checks only MDO (Safe Attachment and Safe Links) policies for user(s) specified in the CSV file or EmailAddresses parameter.
+ShowDetailedPolicies | In addition to the policy applied, show any policy details that are set to True, On, or not blank.
+SkipConnectionCheck | Skips connection check for Graph and Exchange Online.
+ScriptUpdateOnly | Just updates script version to latest one.
+
+## Common Usage
+After downloading the script, you must be connected to both Exchange Online and Microsoft Graph PowerShell to run it.
+
```

Comment:
Can we add some more context here about the requirements and where they can be found?
For example, add the link to the `ExchangeOnlineManagement` PS Gallery site: https://www.powershellgallery.com/packages/ExchangeOnlineManagement

Describe which modules of the PowerShell Graph SDK are at least required to run the script. The PowerShell Graph SDK is very big. You can find the module which contains a PowerShell Graph cmdlet atop the cmdlet documentation.

Example:
https://learn.microsoft.com/powershell/module/microsoft.graph.applications/update-mgserviceprincipal?view=graph-powershell-1.0

`Install-Module -Name Microsoft.Graph.Applications` would do the job here.